mods/stairs/init.lua

```diff
@@ -256,355 +373,425 @@ if replace then
 end
 
 
--- Stair/slab registration function.
--- Nodes will be called stairs:{stair,slab}_<subname>
+-- Stair/slab/stairi/stairo registration function.
+-- Nodes will be called stairs:{stair,slab,stairi,stairo}_<subname>
 
-function stairs.register_stair_and_slab(subname, recipeitem,
-		groups, images, desc_stair, desc_slab, sounds)
+function stairs.register_stair_and_slab_and_stairi_and_stairo(subname, recipeitem,
+		groups, images, desc_stair, desc_slab, desc_stairi, desc_stairo, sounds)
 	stairs.register_stair(subname, recipeitem, groups, images, desc_stair, sounds)
 	stairs.register_slab(subname, recipeitem, groups, images, desc_slab, sounds)
+    stairs.register_stairi(subname, recipeitem, groups, images, desc_stairi, sounds)
+	stairs.register_stairo(subname, recipeitem, groups, images, desc_stairo, sounds)
 end
 
 
--- Register default stairs and slabs
+-- Register default stairs and slabs and corners
 
-stairs.register_stair_and_slab(
+stairs.register_stair_and_slab_and_stairi_and_stairo(
 	"wood",
 	"default:wood",
 	{choppy = 2, oddly_breakable_by_hand = 2, flammable = 2},
 	{"default_wood.png"},
 	"Wooden Stair",
 	"Wooden Slab",
+	"Wooden Stairi",
+	"Wooden Stairo",
```

Comment:
sorry, stairi, and stairo refer to inner and outer. this is the way the original author used descriptions. I can change to inner_stair & outer_stair is prefered